### PR TITLE
HW-54157: Add getField(String) in HyperwalletTransferMethod and HyperwalletUser

### PR DESCRIPTION
**Pull request by oleksii-lubianyi-epam**

### CODE REVIEW

File: Sources/Model/TransferMethod/HyperwalletTransferMethod.swift [hunk 1]
```diff
@@ -178,10 +178,18 @@ public class HyperwalletTransferMethod: Codable {
     ///
     /// - Parameter fieldName: The `TransferMethodField` type
     /// - Returns: Returns the field value, or nil if none exists.
-    public func getField(fieldName: TransferMethodField) -> Any? {
+    private func getField(fieldName: TransferMethodField) -> Any? {
```
Comment (jsaini-hw) [inline Sources/Model/TransferMethod/HyperwalletTransferMethod.swift]: remove this method

File: Sources/Model/User/HyperwalletUser.swift [hunk 2]
```diff
@@ -447,10 +447,18 @@ public struct HyperwalletUser: Codable {
     ///
     /// - Parameter fieldName: The `UserField` type
     /// - Returns: Returns the field value, or nil if none exists.
-    public func getField(_ fieldName: UserField) -> Any? {
+    private func getField(_ fieldName: UserField) -> Any? {
```
Comment (jsaini-hw) [inline Sources/Model/User/HyperwalletUser.swift]: remove this method

File: Sources/Model/TransferMethod/HyperwalletBankAccount.swift [hunk 3]
```diff
@@ -120,7 +120,9 @@ public final class HyperwalletBankAccount: HyperwalletTransferMethod {
 
     /// The user's relationship with the bank account holder.
     public var bankAccountRelationship: RelationshipType? {
-        guard let relationship = getField(fieldName: .bankAccountRelationship) as? String else {
+        let relationshiFieldName = TransferMethodField.bankAccountRelationship.rawValue
```
[the remark this answers is not in the document]
Comment (oleksii-lubianyi-epam) [reply Sources/Model/TransferMethod/HyperwalletBankAccount.swift]: Thanks, fixed

File: Sources/Model/TransferMethod/HyperwalletTransferMethod.swift (hunk 1 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Model/TransferMethod/HyperwalletTransferMethod.swift]: Done

File: Sources/Model/User/HyperwalletUser.swift (hunk 2 above)
Comment (oleksii-lubianyi-epam) [reply Sources/Model/User/HyperwalletUser.swift]: Done

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift [hunk 4]
```diff
@@ -658,4 +669,22 @@ private extension HyperwalletBankAccountIndividualTests {
         let responseRole = bankAccountResponse?.businessContactRole?.rawValue
         XCTAssertEqual(responseRole, role.rawValue)
     }
+
+    func verifyBusinessType(_ type: HyperwalletBankAccount.BusinessType,
```
Comment (jsaini-hw) [inline Tests/TransferMethod/HyperwalletBankAccountTests.swift]: make it private

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift [hunk 5]
```diff
@@ -658,4 +669,22 @@ private extension HyperwalletBankAccountIndividualTests {
...
+                            in bankAccountResponse: HyperwalletBankAccount?) {
+        let responseBusinessType = bankAccountResponse?.businessType?.rawValue
+        XCTAssertEqual(responseBusinessType, type.rawValue)
+    }
+
+    func verifyGender(_ gender: HyperwalletBankAccount.Gender,
```
Comment (jsaini-hw) [inline Tests/TransferMethod/HyperwalletBankAccountTests.swift]: make it private

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift [hunk 6]
```diff
@@ -658,4 +669,22 @@ private extension HyperwalletBankAccountIndividualTests {
...
+                      in bankAccountResponse: HyperwalletBankAccount?) {
+        let responseGender = bankAccountResponse?.gender?.rawValue
+        XCTAssertEqual(responseGender, gender.rawValue)
+    }
+
+    func verifyGovernmentIdType(_ governmentIdType: HyperwalletBankAccount.GovernmentIdType,
```
Comment (jsaini-hw) [inline Tests/TransferMethod/HyperwalletBankAccountTests.swift]: make it private

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift (hunk 4 above)
Comment (oleksii-lubianyi-epam) [reply Tests/TransferMethod/HyperwalletBankAccountTests.swift]: These methods have been wrapped in 
```
private extension HyperwalletBankAccountIndividualTests {
...
}
```

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift (hunk 5 above)
Comment (oleksii-lubianyi-epam) [reply Tests/TransferMethod/HyperwalletBankAccountTests.swift]: These methods have been wrapped in 
```
private extension HyperwalletBankAccountIndividualTests {
...
}
```

File: Tests/TransferMethod/HyperwalletBankAccountTests.swift (hunk 6 above)
Comment (oleksii-lubianyi-epam) [reply Tests/TransferMethod/HyperwalletBankAccountTests.swift]: These methods have been wrapped in 
```
private extension HyperwalletBankAccountIndividualTests {
...
}
```

**Pull request merged**